pages/test_analytics/test_ownership.md

```diff
@@ -0,0 +1,111 @@
+# Test ownership
+
+Customers on the [Pro and Enterprise plans](https://buildkite.com/pricing) can assign test ownership to [Teams](/docs/team-management/permissions).
+
+Test ownership is assigned to the tests in a specific suite per instructions in your TESTOWNER file. The team that is the default owner of a test [will be auto-assigned to fix any flakes](/docs/test-analytics/flaky-test-assignment) that occur.
+
+> 🚧 Buildkite test ownership is currently in private beta
+> Please reach out to our support team to register for early access.
+
+## Setting test ownership
+
+You can upload a TESTOWNER file via this API endpoint:
+
+```bash
+curl --location 'https://analytics-api.buildkite.com/v1/test-ownerships' \
+     --header "Authorization: Bearer <your-suite-api-token>" \
+     -F 'file=@<your-TESTOWNER-file-location>'
+```
+You might consider creating a new pipeline to automatically upload your TESTOWNER file when changes are detected.
+
+## Your TESTOWNER file
+
+A TESTOWNER file [follows the same rules as a `.gitignore` or `CODEOWNERS` file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file), with one exception.
```

Comment:
it would be cool if we could link to the one-exception and put it at the bottom of the doc under some kinda troubleshooting section, that way we get to keep them on the happy path.

Reply:
Yeah I think this section could just be moved down now as we have the example TESTOWNER file that describes the rules specifically now anyway.